File: codeforces/iq_test.cpp
#include <bits/stdc++.h>

using namespace std;

static int integers[101]{0};

int main()
{
  int n;
  cin >> n;
  pair<int, int> odd{0, 0}, even{0, 0};
  for (int i{0}; i < n; i++)
  {
    cin >> integers[i];
    if (integers[i] & 1)
    {
      odd.first++;
      odd.second = i + 1;
    }
    else
    {
      even.first++;
      even.second = i + 1;
    }
  }

  cout << (odd.first == 1 ? odd.second : even.second) << "\n";

  return 0;
}

File: codeforces/iq_test.js
// const { readline, print } = require('@ip-algorithmics/codeforces-io');

var n = parseInt(readline());
var integers = new Array(101).fill(0);
var odd = [0, 0];
var even = [0, 0];
var integers = readline().split(" ").map(x => parseInt(x))

for (var i = 0; i < n; i++) {
  if (integers[i] % 2 !== 0) {
    odd[0]++;
    odd[1] = i + 1;
  } else {
    even[0]++;
    even[1] = i + 1;
  }
}

print(odd[0] == 1 ? odd[1] : even[1]);
